Index: src/Homework4B.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Homework4B.java b/src/Homework4B.java
new file mode 100644
--- /dev/null	
+++ b/src/Homework4B.java	
@@ -0,0 +1,15 @@
+/**
+ * Homework for arrays class vol 2
+ *
+ * @author Kätlin Padesaar-Korela
+ */
+
+public class Homework4B {
+    public static void main(String[] args) {
+
+        //Write a Java program to calculate the average value of array elements: [1, 7, 3, 10, 9]
+
+        int[] a = {1, 7, 3, 10, 9 }
+
+    }
+}
